### PR TITLE
SUpport running raw docker servers directly and avoid truss

### PR DESCRIPTION
**Pull request by bolasim**

### CODE REVIEW

File: truss/templates/docker_server/proxy.conf.jinja [hunk 1]
```diff
@@ -0,0 +1,31 @@
+server {
+    # We use the proxy_read_timeout directive here (instead of proxy_send_timeout) as it sets the timeout for reading a response from the proxied server vs. setting a timeout for sending a request to the proxied server.
+    listen 8080;
+
+    # Liveness
+    location / {
```
Comment (vshulman) [inline truss/templates/docker_server/proxy.conf.jinja:6]: remind me the function of readiness vs. liveness?

Comment (amiruci) [reply truss/templates/docker_server/proxy.conf.jinja:6]: i've had to look [this](https://chatgpt.com/share/e/ed536f84-c582-4dd0-8ae6-a1e8318c19d9) up a few times.

File: truss/truss_config.py [hunk 2]
```diff
@@ -506,6 +531,7 @@ class TrussConfig:
     # spec_version is a version string
     spec_version: str = DEFAULT_SPEC_VERSION
     base_image: Optional[BaseImage] = None
+    docker_server: Optional[DockerServer] = None
```
Comment (squidarth) [inline truss/truss_config.py:534]: let's call this custom_server?

**Comment by bolasim** [comment]
Closing in favor of tianshu's pr

**Pull request closed without merging**